Managers/PlayerManager.gd

```diff
@@ -5,12 +5,21 @@ extends Node
 ## PlayerManager.player.
 
 
-var player: Player
-
 signal on_player_initialized
 
+var player: Player
+var player_persistent_data: PlayerPersistentData = null
+
 
 func set_player(in_player: Player) -> void:
 	player = in_player
 	if player != null:
 		on_player_initialized.emit()
+
+
+func create_persistent_data() -> void:
+	PlayerManager.player_persistent_data = PlayerPersistentData.new()
+
+
+func get_persistent_data() -> PlayerPersistentData:
```

Comment:
could use a getter directly with the declaration of the variable (same as another comment I made)

Reply:
In that case the syntax would be `PlayerManager.player_persistent_data` when you want to get the data right? Is there any benefit to doing that?